<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="py_QWidget" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidgetLocations">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <column>
        <property name="text">
         <string>Location</string>
        </property>
       </column>
      </widget>
      <widget class="QTreeWidget" name="treeWidgetDevices">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>Device</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address:Port</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidgetTests">
       <column>
        <property name="text">
         <string>Test</string>
        </property>
       </column>
      </widget>
      <widget class="QTreeWidget" name="treeWidgetModels">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Model</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>File Path</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionExpand">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/go-down.png</normaloff>:/test/icons/go-down.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Expand</string>
   </property>
   <property name="toolTip">
    <string>Expand tests</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/go-bottom.png</normaloff>:/test/icons/go-bottom.png</iconset>
   </property>
   <property name="text">
    <string>Expand &amp;all</string>
   </property>
   <property name="toolTip">
    <string>Expand all tests</string>
   </property>
  </action>
  <action name="actionCollapse">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/go-up.png</normaloff>:/test/icons/go-up.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Collapse</string>
   </property>
   <property name="toolTip">
    <string>Collapse tests</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/go-top.png</normaloff>:/test/icons/go-top.png</iconset>
   </property>
   <property name="text">
    <string>Co&amp;llapse all</string>
   </property>
   <property name="toolTip">
    <string>Collapse all tests</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/view-refresh.png</normaloff>:/test/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh view</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/media-playback-start.png</normaloff>:/test/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start tests</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/media-playback-stop.png</normaloff>:/test/icons/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top</string>
   </property>
   <property name="toolTip">
    <string>Stop tests</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/media-playback-pause.png</normaloff>:/test/icons/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause tests</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/archive-insert.png</normaloff>:/test/icons/archive-insert.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="toolTip">
    <string>Add location</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/archive-remove.png</normaloff>:/test/icons/archive-remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove location</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="../../icons/test/icons.qrc">
     <normaloff>:/test/icons/media-playback-start.png</normaloff>:/test/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Res&amp;ume</string>
   </property>
   <property name="toolTip">
    <string>Resume tests</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>py_QWidget</class>
   <extends>QWidget</extends>
   <header>py_qwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/test/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
